docs/datatypes.rst

```diff
@@ -0,0 +1,65 @@
+.. _edgedb_java_datatypes:
+
+=========
+Datatypes
+=========
+
+The java binding maps the following EdgeDB types to the corresponding JVM types:
```

Comment:
Line was a character too long, but I think we can just lose the "the" and it still reads fine.
```suggestion
The Java binding maps the following EdgeDB types to corresponding JVM types:
```